javascript/kcc-issuance-snippet/README.md

```diff
@@ -0,0 +1,109 @@
+# KCC Issuance Snippet
+
+This is a snippet to illustrate issuing a [Known Customer Credential](https://github.com/TBD54566975/known-customer-credential) using Web5.
+
+
+> [!NOTE]
+> 
+> This example focuses solely on issuing the KCC. If you'd like an example that shows the entire Identity Verification (IDV) flow, see the [kcc-prototype-exemplar](https://github.com/TBD54566975/tbd-examples/tree/main/javascript/kcc-prototype-exemplar).
+
+## Pre-reqs
+1. [Create DID for Issuer](https://developer.tbd.website/docs/web5/decentralized-identifiers/how-to-create-did)
+2. Obtain/create DID URI for Customer
+3. [Install `web5/credentials`](https://www.npmjs.com/package/@web5/credentials)
+
+## Create Verifiable Credential (VC)
+
+```js
+const known_customer_credential = await VerifiableCredential.create({
```

Comment:
wondering if it's worth it to put 
`import { VerifiableCredential } from "@web5/credentials";`

or if that's excessive. I know the npmjs link above shows import for VC, VP, and PEX..but maybe just adding for clarity